runelite-client/src/main/java/net/runelite/client/plugins/music/MusicPlugin.java

```diff
@@ -590,4 +621,15 @@ else if (source == null
 			areaSoundEffectPlayed.consume();
 		}
 	}
+
+	@Subscribe
+	public void onSoundEffectPlayed(SoundEffectPlayed soundEffectPlayed)
+	{
+		int soundId = soundEffectPlayed.getSoundId();
```

Comment:
This value is used only once, and is clear without a variable name attached to it, so this should be passed directly to the `contains()` check instead.